File: tests/fixtures/real_world/php/taint/data_exfil_user_input_silenced.php
<?php

function forward_message() {
    $msg = $_POST['message'];
    $ch = curl_init('https://telemetry.internal/forward');
    curl_setopt($ch, CURLOPT_POSTFIELDS, "message={$msg}");
    curl_exec($ch);
    curl_close($ch);
}

File: tests/fixtures/real_world/c/taint/data_exfil_user_input_silenced.c
#include <curl/curl.h>
#include <stdio.h>

void forward_stdin() {
    char input[256];
    if (!fgets(input, sizeof(input), stdin)) return;

    CURL *curl = curl_easy_init();
    curl_easy_setopt(curl, CURLOPT_URL, "https://telemetry.internal/forward");
    curl_easy_setopt(curl, CURLOPT_POSTFIELDS, input);
    curl_easy_perform(curl);
    curl_easy_cleanup(curl);
}
